ingestion/ingestion_dependency.sh

```diff
@@ -39,6 +39,9 @@ airflow users create \
 
 (sleep 5; airflow db upgrade)
 (sleep 5; airflow db upgrade)
+
+chmod 666 /var/run/docker.sock
```

Comment:
Hello @pmbrull . This might not work with k8s as there is now `/var/run/docker.sock` file! we might want to use this explicitly with docker-compose files using command and arguments!

Reply:
oh actually this one I needed to remove hehe thanks, missed it!

Reply:
that was some old testing trying to prep the permissions on the volume and here it does not work. Instead, added the permission on the docker-compose itself